src/databases/mongo.database.ts

```
@@ -417,6 +408,10 @@
 	async updateOne(options: DatabaseUpdateOneOptions, x_request_id: string): Promise<FindOneResponseObject> {
 		const mongo = await this.createConnection(options.schema.table)
 
+		if (options.data['_id']) {
+			delete options.data['_id']
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)